docs/application/vscode-ext/web.md

```diff
@@ -0,0 +1,306 @@
+  2. Select the profiles and versions that you want to install and click **Ok**.
+
+     ![Emulator Manager](media/howtoinstall-emulatormanager.png)
+
+
+## Develop Applications
+
+The following section explains how to use VS Code Extension for Tizen Web to develop the applications:
+
+### Create Tizen Web Project
+
+To create a Tizen Web project, follow these steps:
+
+1. Open **Command Palette** and enter **Create Wizard Project**.
+
+   ![Create Wizard Project](media/tizen_web7.png)
+
+2. Select the **Template** project type and click **Next**.
+
+   ![Select the type of project](media/tizen_web8.png)
+
+3. Select the application profile and version from the drop-down list and click Next.
+
+   ![Select the profile and version](media/tizen_web9.png)
+
+4. Select the **Basic UI** template and click **Next**.
+
+  > **Note**
+  >
+  > The list is different depending on the type and profile of the previously selected project.
+
+  ![Select the project](media/tizen_web10.png)
+
+5. Enter the **Project name** and click **Finish** to complete the project creation in workspace.
+  > **Note**
+  >
+  > packageId is an optional field, it is not mandatory to enter details in this field because the packageId will generate the details automatically.
+
+  ![Enter the name of project](media/tizen_web11.png)
+
+6. If you checked all this, the project is created.
+
+   ![Project is created](media/tizen_web12.png)
+
+### Creating Tizen Certificate
+
+To create Tizen Certificate, follow these steps:
+
+  1. Open **Command Palette** and enter **Create Tizen Certificate**.
+  2. Enter the name of the certificate.
+  3. Enter the password of the certificate.
+  4. Enter the profile name of the certificate. (Extension is .p12)
+  > **Note**
+  > 
+  > If it is created properly, profilename.p12 will be created in tizen-studio/keystore/author.
+
+To add Tizen Certificate, follow these steps:
+
+  1. Open **Command Palette** and enter **Add Tizen Certificate**.
+  2. Enter the name of the certificate.
+  3. Enter the password of the certificate.
+  4. Enter the profile name of the certificate. (Extension is .p12)
+  > **Note**
+  > 
+  > You can verify the added certificate by running **Certificate Manager**.
+
+To Remove your Tizen certificate:
+
+If you want to change the Certificate, you need to remove the added Certificate.
+
+  1. Open **Command Palette** and enter **Remove Tizen Certificate**.
+  2. Enter the name of the certificate.
+  > **Note**
+  >
+  > When you run Certificate Manager the certificate will be deleted.
+
+### Building Project
+
+To build project:
+
+  1. Open **Command Palette** and enter **Build Tizen App**.
+  2. Enter the folder name of the project you want to build. (Folder name, not path)
+  3. When the build is complete, it succeeds, and there is successful generation of wgt package popup. Also, you will see a build success log in the terminal at the bottom on the vscode.
```

Comment:
@mijin85cho  and @DohyungLim
When the build is successfully completed, the wgt packaging dialog is generated. 

"wgt packaging" is the name of the dialog?

Reply:
@annie-samsung Would you please also see this with @sabeena21?

Reply:
@mijin85cho, @sabeena21 
My opinion: It is better we do not refer to the notification as a "dialog" or "popup". Instead just say "notification appears".
https://code.visualstudio.com/updates/v1_21#_workbench


Here is my recommendation for all 3 cases:
_____________________________________________

When the app is successfully built, “WGT Packaging Success”  and “Build Success” notifications appear. Also, you will see a "BUILD SUCCESSFUL" log in the **TERMINAL** panel.

When the app is successfully installed, “Installation App Successfully” notification appears. Also, you will see a "BUILD SUCCESSFUL" log in the **TERMINAL** panel.

When the project execution is successfully completed, “Launch App Successfully” notification appears. Also, you will see a "BUILD SUCCESSFUL" log in the **TERMINAL** panel.
_____________________________________________

@DohyungLim,
1) I suppose, the log appears in the OUTPUT panel and not in TERMINAL, could you confirm that we should say the log is visible in TERMINAL panel?

2) Also, the UI messages need to rephrased, as mentioned earlier it would be good get all UI messages in the VS code extension reviewed for language:

Installation App Successfully >>> App is successfully installed.
Launch App Successfully>>>App is successfully launched.

Also, in some places S of Success is in caps, some places in lower case.

Reply:
@annie-samsung As you commented here, the messages are need to be fixed, which can allude the possibility of sentence changing. 
How about using vague description for the sentence itself, not using the exact sentence?
For example, the original sentence was "the wgt packaging successful popup is generated"

Reply:
Please review the document first.
The contents of the UI message will be modified separately.